notebook_data_redirector/common.py

```diff
@@ -141,12 +182,13 @@ def iterate_files(folder):
                 # Here we're recursively calling iterate_files on a nested folder and
                 # receiving an iterator that contains all of its files.  "yield from"
                 # will yield each value from that iterator in turn.
-                yield from iterate_files(item)
+                yield from iterate_files(item, shared=shared or is_box_file_public(item.get()))
```

Comment:
Did it prove necessary to call `get()`?  The item should already include the shared_link object since we requested it in the get_items query.

Reply:
nice catch, doesn't seem to be needed.

Reply:
tested via IPython using the real boxsdk